samples/uikit/README.md

```diff
@@ -5,22 +5,14 @@ Apple devices, such as an iPhone.
 
 To build and run the sample do the following:
 
-1.  Open `samples/uikit/konan.xcodeproj` with Xcode.
+1.  Open  `UIKitSample.xcodeproj` with Xcode and fix bundle ID and development team.
+  or
+1a.  Fix bundleIdPrefix and DEVELOPMENT_TEAM in project.yml and generate XCode project
+    with xcodegen (https://github.com/yonaskolb/XcodeGen/).
 
-2.  Open the project's target through project navigator, go to tab 'General'.
-    In 'Identity' section change the bundle ID to the unique string in
-    reverse-DNS format. Then select the team in 'Signing' section.
+2.  Now build and run the application on a connected iPhone with Xcode or simulator.
```

Comment:
Something wrong.
Is the application supposed to be run on simulator without Xcode?

Reply:
I meant iWatch, not XCode :). Will fix.